adafruit_progressbar/verticalprogressbar.py

```diff
@@ -0,0 +1,173 @@
+# SPDX-FileCopyrightText: Copyright (c) 2020-2021 Brent Rubell for Adafruit Industries, Hugo Dahl
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`verticalprogressbar`
+================================================================================
+
+Dynamic progress bar widget for CircuitPython displays
+
+
+* Author(s): Brent Rubell, Hugo Dahl
+
+Implementation Notes
+--------------------
+
+**Software and Dependencies:**
+
+* Adafruit CircuitPython firmware for the supported boards:
+  https://github.com/adafruit/circuitpython/releases
+
+"""
+
+# imports
+from enum import Enum
+from . import ProgressBarBase
+
+
+class VerticalFillDirection(Enum):
+    """This enum is used to specify the direction in which the progress
+    bar's display bar should fill as the value represented increases."""
+
+    BOTTOM_TO_TOP = 0, "Fills from the bottom up toward the top"
+    DEFAULT = BOTTOM_TO_TOP, "Default fill direction (BOTTOM_TO_TOP)"
+    TOP_TO_BOTTOM = 1, "Fills from the top down toward the bottom"
+
+
+# pylint: disable=too-many-arguments, too-few-public-methods, too-many-instance-attributes
+class VerticalProgressBar(ProgressBarBase):
+    """A dynamic progress bar widget.
+
+    The anchor position is the position where the control would start if it
+    were being read visually or on paper, where the (0, 0) position is
+    the lower-left corner for ascending progress bars (fills from the bottom to
+    to the top in vertical bars, or from the left to the right in horizontal
+    progress bars), upper-left corner for descending progress bars (fills from
+    the top to the bottom).
+
+    Using the diagrams below, the bar will fill in the following directions::
+
+                         -----------------------------
+                         | Horizontal   | Vertical   |
+        ----------------------------------------------
+        | Ascending      |  1-3 to 2-4 |  3-4 to 1-2 |
+        ----------------------------------------------
+        | Descending     |  2-4 to 1-3 |  1-2 to 3-4 |
+        ----------------------------------------------
+
+        Vertical            Horizontal
+
+        1--2                1-----------------------2
+        |  |                |                       |
+        |  |                |                       |
+        |  |                3-----------------------4
+        |  |
+        3--4
+
+    :param anchor_position: The anchor coordinates of the progress bar.
```

Comment:
I'm not sure if it causes trouble with inheritance, but if not. I think we should make this parameter name match the same parameter in the ProgressBarBase class. So name this one `position` instead of `anchor_position`. This will help consistency, and leave the door open for us for the future PR to and anchor_point and position.

Reply:
Good point. Will make that change. My original "way back when" plan, before learning about the other anchor_point, which is probably based on the same intention!